### PR TITLE
Update @types/jquery and @types/node

### PR DESCRIPTION
**Pull request by jamesnakagawa**
Tests now run without error as of 2f6eebf9abd0069327b724bb2923757be65b9b4f.

### CODE REVIEW

**Comment by donkirkby** [comment]
@jamesnakagawa, does this mean we should upgrade from Node 6 to Node 12?

**Comment by jamesnakagawa** [comment]
Yes! Since Node is only used in the backend, the change will only affect the test runner and the webpack step. As well, we're not using any of the Node-only features, so the risk of regression is fairly low. (And of course, the tests all ran.)

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ before_install:
 install:
     - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then pip install -r requirements-dev.txt ; fi
     - if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pip install -r requirements-dev.py34.txt ; fi
-    - curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+    - curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
```
Comment (donkirkby) [inline .travis.yml]: If you look in the Travis build log, it looks like there's another step required after this one. It's still running node 8, not 12.

**Comment by donkirkby** [comment]
I just noticed that nodejs is getting installed in both the `before_install` phase and the `install` phase. I'm not sure what the difference between the phases is, but we probably don't need to install it twice.

**Comment by jamesnakagawa** [comment]
Good eye. It's still using Node 8, access as `node`, while Node 12 is installed as `nodejs`.

Is there an easy way to test this locally on my machine? If not I can continue to spam this branch... :sweat_smile:

**Comment by donkirkby** [comment]
I think there's a docker image for Travis builds, if you want to hunt that down, but feel free to spam the branch. Would using `nvm` help?

If you run `npm version` instead of `npm --version`, it tells you what version of node it's using.

**Comment by jamesnakagawa** [comment]
Yep, I just realized that the docker image comes preinstalled with `nvm` and a whole bunch of goodies. So that's where Node v8 was coming from.

**Comment by jamesnakagawa** [comment]
Somewhat worrying is that Canvas-for-Node is not compiling correctly. However, have we successfully rid ourselves of all Canvas-based illustrations in Kive? If so we can get rid of the dependency.

**Comment by donkirkby** [comment]
I think we're still using canvas in the [archive containers].

[archive containers]: https://github.com/cfe-lab/Kive/blob/master/kive/container/static/container/canvas/drydock_objects.ts#L17

**Comment by jamesnakagawa** [comment]
Okay, I'll dig into this. Currently I'm not sure our imagediff-based canvas tests are working at all (as in, they might be false positives)

**Comment by jamesnakagawa** [comment]
Fixed by upgrading node-sass and also upgrading canvas in a forked version of imagediff.

**Pull request merged**